src/UI/Component/Button/Split/Factory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Button\Split;
+
+/**
+ * This is how a factory for split buttons looks like.
+ */
+interface Factory
+{
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Standard Split Buttons is used to group a list of actions with one labeled as default.
+	 *   composition: >
+	 *       The Standard Split Button shows the options in the dropdown as simple list.
+	 *   effect: >
+	 *       Selecting one of the actions in the dropdown directly fires it.
+	 *
+	 * rules:
+	 *   interaction:
+	 *       1: >
+	 *          Selecting one of the actions in the dropdown of the Split Button MUST directly fire this action.
+	 * ---
+	 * @param    array $labelled_actions Set of labelled actions (string|string)[]. The label of the action is used as key, the action itself as value.
+	 *          The first of the actions will be used as default action, directly visible.
+	 *
+	 * @return  \ILIAS\UI\Component\Button\Split\Standard
+	 */
+	public function standard($labelled_actions);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Month Split Button enables to select a specific month to fire some action (probably a change of view).
+	 *   composition: >
+	 *       The Month Split Button is composed of a Button showing the default month directly (probably the
+	 *       month currently rendered by some view). The Dropdown contains an interface enabling the selection of a month from
+	 *       the future or the past.
+	 *   effect: >
+	 *      Selecting a month from the dropdown directly fires the according action (e.g. switching the view to the selected month).
+	 *
+	 * rules:
+	 *   interaction:
+	 *       1: >
+	 *          Selecting a month from the dropdown MUST directly fire the according action.
```

Comment:
You have a valid point there. Maybe the issue here is what we display in the button. How about changing it from displaying the current month to "Today". Clicking it would always show the current month. 

Used along with the Section View Control of https://github.com/ILIAS-eLearning/ILIAS/pull/549 we would get:
< Today v >

Obviously this implies, that we would have to put the information "November 2016" (the currently displayed month somewhere else, probably either on the left, right or bellow the control (that's for Yvonne to decide). See: http://www.ilias.de/docu/goto_docu_wiki_wpage_4410_1357.html

@alex40724 what do you think?